lxd/api.go

```diff
@@ -109,7 +109,8 @@ func restServer(d *Daemon) *http.Server {
 		uiHandlerErrorUINotEnabled := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 			w.Header().Set("Content-Type", "text/html")
 			w.WriteHeader(http.StatusServiceUnavailable)
-			fmt.Fprint(w, errorMessage)
+			_, err := fmt.Fprint(w, errorMessage)
+			logger.Error(fmt.Sprintf("Could not write error message to writer object: %s", err.Error()), logger.Ctx{"url": r.URL, "method": r.Method, "remote": r.RemoteAddr})
```

Comment:
Please use %v and just pass in err, rather than manually converting to Error string.